### DISCUSSION TITLE
How to Toast with NextJS 13?

### DISCUSSION THREAD

**Discussion by w-dib** [timolins/react-hot-toast#263] [Q&A]
Hi! When trying to use the react-hot-toast <Toaster/> component in app/layout.jsx, I get trouble because NextJS 13 defaults to SSR.

What's the best way around that? Where should I place the <Toaster/> component if I wish to use it across the website?

Thanks!

**Comment by ryandotelliott** [comment]
I placed my component in a child `layout.tsx` which is using `"use client"`.

**Comment by timolins** [comment]
Good news, you can now update to https://github.com/timolins/react-hot-toast/releases/tag/v2.4.1 which includes "use client" – so no additional work is required. 🙏
